src/test/fake-config.ts

```diff
@@ -0,0 +1,68 @@
+import { injectable } from 'inversify';
+import { ConfigLoader } from '../types';
+
+@injectable()
+export class FakeConfigManager implements ConfigLoader {
+
+    private _mode: string;
+    private _dbMode: string
+    private _dbHost: string;
+    private _dbPort: number;
+    private _dbUser: string;
+    private _dbName: string;
+    private _dbPass: string;
+
+    public get mode(): string {
```

Comment:
you could simply make the field public without explicitly using getter and setter.